<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="lovepreet" name="TestClasses.LoginUrbanLadderTest" tests="2" failures="0" timestamp="1 Jun 2020 07:11:14 GMT" time="68.884" errors="1">
  <testcase name="LoginToUrbanLadder" time="39.650" classname="TestClasses.LoginUrbanLadderTest"/>
  <testcase name="LoginToUrbanLadder" time="29.234" classname="TestClasses.LoginUrbanLadderTest">
    <error type="junit.framework.AssertionFailedError" message="Username or email is incorrect">
      <![CDATA[junit.framework.AssertionFailedError: Username or email is incorrect
	at junit.framework.Assert.fail(Assert.java:47)
	at BaseClasses.PageBaseClass.reportFail(PageBaseClass.java:76)
	at PageClasses.ObjectSelection.Logincheck(ObjectSelection.java:29)
	at TestClasses.LoginUrbanLadderTest.LoginToUrbanLadder(LoginUrbanLadderTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- LoginToUrbanLadder -->
</testsuite> <!-- TestClasses.LoginUrbanLadderTest -->
